adobe/spectrum-css
# fix(meter): update layout tokens

## Description



1. Updates Meter to use new layout tokens:
`meter-minimum-width`
`meter-maximum-width`
`meter-default-width`
`meter-thickness-small`
`meter-thickness-large`
Large value from `component-top-to-text-200` to `component-top-to-text-75` 

2. Add `Meter` Stories 
3. Added controls to Progress Bar: value and label

Bit of context, for now `Meter` uses `ProgressBar` css. 

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.



#### Test Outline 
 1. Open the [storybook](https://pr-2080--spectrum-css.netlify.app/preview/?path=/story/components-meter) for the Meter component:
- [x] Ensure Meter displays appropriately 
- [x] Toggle between fill colors 
- [x] toggle between meter sizing [@mlogsdon18]

2. Open the [storybook](https://pr-2080--spectrum-css.netlify.app/preview/?path=/story/components-progress-bar) for the Progress Bar component:
- [x] Ensure controls all work as they should [@mlogsdon18]

3. Open the [Docs site](https://pr-2080--spectrum-css.netlify.app/meter) for the Meter component. 
- [x] inspect for new tokens 

### Regression testing

Validate:

1. A legacy documentation page (such as [accordion](https://pr-###--spectrum-css.netlify.app/accordion.html)), including:


4. A migrated documentation page (such as [action group](https://pr-###--spectrum-css.netlify.app/actiongroup.html)), including:


## Screenshots



### Before
![Screenshot 2023-08-10 at 4 08 49 PM](https://github.com/adobe/spectrum-css/assets/63808889/760bac21-7f31-4e35-a233-d3b0fe9bc8ed)

### After
![Screenshot 2023-08-10 at 4 09 18 PM](https://github.com/adobe/spectrum-css/assets/63808889/778e603f-b7b2-4c01-b0f5-38b64d06df8a)

## To-do list
